samples/Builder.Filtering.Web/project.json

```diff
@@ -1,14 +1,16 @@
 {
   "dependencies": {
     "Microsoft.AspNet.Http": "1.0.0-*",
-    "Microsoft.AspNet.Server.IIS" : "1.0.0-*",
-    "Microsoft.AspNet.Server.WebListener" : "1.0.0-*"
+    "Microsoft.AspNet.Server.IIS": "1.0.0-*",
+    "Microsoft.AspNet.Server.Kestrel": "1.0.0-*",
+    "Microsoft.AspNet.Server.WebListener": "1.0.0-*"
   },
   "frameworks": {
     "dnx451": { },
     "dnxcore50" : { }
   },
   "commands": {
```

Comment:
Change the commands to 
```
    "web": "Microsoft.AspNet.Server.Kestrel",
    "weblistener": "Microsoft.AspNet.Server.WebListener"
```
Compare to https://github.com/aspnet/Security/pull/597/files#diff-68483d081e22a22c9f6fa534b0a1abea